README.md

```diff
@@ -67,14 +66,15 @@ Got any questions? Want to talk to the maintainers?
 Our [Discord community](https://discord.gg/2QDjZkYunf) is a great place to get in touch with us, and
 we'd love to have you there.
 
-<a href="https://discord.gg/2QDjZkYunf" alt="Melt UI Discord community">
-<picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://invidget.switchblade.xyz/2QDjZkYunf">
-  <img alt="Melt UI Discord community" src="https://invidget.switchblade.xyz/2QDjZkYunf?theme=light">
-</picture>
-</a>
+[![Melt UI Discord community](https://invidget.switchblade.xyz/2QDjZkYunf?theme=dark)](https://discord.gg/2QDjZkYunf)
 
 ## Similar projects
 
 Looking for more? Check out the
 [other component library projects available for Svelte](https://sveltesociety.dev/components#design-systems).
+
+## Thanks to all Contributors
+
+Thank you very much for taking your time to help Melt.
+
+[![Contributors](https://contrib.rocks/image?repo=melt-ui/melt-ui)]([https://github.com/codemaniac-sahil/news-webapp-api](https://github.com/melt-ui/melt-ui)https://github.com/melt-ui/melt-ui/graphs/contributors)
```

Comment:
I like this section! Although maybe it can be moved. Instead of having its own headline, we can add the text and image on the bottom of the `Contributing` section. Also we can change the text: `Melt UI is an open-source project built by the community for the community. It wouldn't be possible if it wasn't for the work of some amazing people.`

Reply:
Resolved in the commit 94c1983aee751d18acd48c5d699e9629612dbd0e

Attached capture:
![image](https://github.com/melt-ui/melt-ui/assets/80181518/856231cc-865e-4fb5-905a-cdcd8cc5f0a8)